### ISSUE TITLE
jsnext:main for @storybook/react is not included in published package

### ISSUE THREAD

**Issue by travi** [storybooks/storybook#2627]
* https://github.com/storybooks/storybook/blob/911e6156f17ebe153418133dfc7d077eb1fb96a2/app/react/package.json#L11
* https://github.com/storybooks/storybook/blob/911e6156f17ebe153418133dfc7d077eb1fb96a2/app/react/.npmignore#L2

**Comment by JonShort** [comment]
Looks like this was addressed in #2604

**Comment by travi** [comment]
any ETA on publishing this? it has my build in a broken state due to eslint trying to follow that reference instead of the `main` that used to work:

```
3:25  error  Unable to resolve path to module '@storybook/react'  import/no-unresolved
```

**Comment by shilman** [comment]
The fix is live now in `@storybook/react@3.3.5`
